packages/application/src/requests.tsx

```diff
@@ -82,7 +109,18 @@ export class RequestHandler {
       displayWaitDialog
     );
   }
-
+  /**
+   * Make an request to the jupyterlab server.
+   *
+   * @param requestExt - The url for the request.
+   *
+   * @param requestOptions - The initialization options for the request.
+   *
+   * @param displayWaitDialog - Whether or not to display a dialog warning
+   * warning the use the request may take time.
+   *
```

Comment:
If we don't make a wrapper, could we make this a more generic name that flags the call as expensive, which will take a while to be submitted, and would for now require the dialog to show up, but could enable other things to be handled in this special case?

Reply:
hmmm, I guess we can. I'll change the name to `longRequest`, it's more generic but still gets its purpose across.